packages/api/cms-api/src/auth/strategies/auth-proxy-jwt.strategy.ts

```diff
@@ -45,12 +47,11 @@ export function createAuthProxyJwtStrategy({
                 audience,
                 ...strategyOptions,
             });
+            if (!this.currentUserLoader) this.currentUserLoader = currentUserLoader ?? new CurrentUserLoader();
```

Comment:
I think the default should be defined in the module config, would that be possible? Or no default at all to make it more explicit. Or is the default a common use case?

Currently you define it in AppModule, shouldn't that be part of a module?

Reply:
42dc30d0f8c2f8ef996358c1efd0c1e8b0f63dd9
The CurrentUserLoader now has a more specific interface and is optional, it should be only set when it is required to really load the currentUser (e.g. when using the UserPermissions-Module).